test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -43,6 +43,8 @@ TAGFILES               += "${SEQAN3_DOXYGEN_STD_TAGFILE}=http://en.cppreference.
 
 EXCLUDE_SYMBOLS        = seqan3::contrib
 
+FORMULA_FONTSIZE       = 14
+
```

Comment:
remove extra new line :nail_care: 
Then we can merge this

Reply:
If you rebase onto the newst master maybe the code coverage CI wont be confused and also be green :roll_eyes:

Reply:
Okay, thanks!